pkg/cmd/auth/auth.go

```diff
@@ -14,8 +14,11 @@ import (
 func NewCmdAuth(f *cmdutil.Factory) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "auth <command>",
-		Short: "Login, logout, and refresh your authentication",
+		Short: "Authenticate gh with GitHub",
```

Comment:
Do we want to mention anything about `setup-git` command in here or the `long` description since it is technically about authenticating `git` with GitHub?